### DISCUSSION TITLE
Need a Front-end tag.

### DISCUSSION THREAD

**Discussion by nguyenthanhan201** [dailydotdev/daily#484] [Feature Requests]
I think we need a front-end tag.

**Comment by idoshamun** [comment]
Hi @NguyenThanhAn201,

What content would you expect to have in the front-end tag?

**Comment by nguyenthanhan201** [comment]
Tips or libraries for front-end websites. And projects that help develop
coding skills.

Vào CN, 20 thg 2, 2022 vào lúc 16:21 Ido Shamun ***@***.***>
đã viết:

**Reply by idoshamun** [reply]
Frontend is a wide topic that goes through web development, mobile development, and more.
This is why I'm trying to understand what frontend is exactly.

**Reply by kenchoong** [reply]
may be like group

 #nextjs, #gatsbyjs, #svelte. #reactjs , #redux  -> #web-development
#android, #ios, #react-native -> #mobile-development 

so we have #web-development and #mobile-development, 2 tags

**Reply by idoshamun** [reply]
So a new category here you mean?
Currently, we have a category for webdev and mobile
<img width="363" alt="image" src="https://user-images.githubusercontent.com/1993245/156507387-0b233efc-1e2e-4e1c-9c77-6229cf796592.png">

**Reply by kenchoong** [reply]
ya.. Like the "backend"

**Reply by idoshamun** [reply]
Although I think frontend as you mentioned is actually split to webdev and mobiledev, don't you think that this is more accurate?

**Reply by kenchoong** [reply]
Just check the daily.dev website, right now I seen `Mobile` and `Webdev`, just very bottom I didnt notice. Sorry 

ya with Mobile and Webdev then should be ok

**Reply by idoshamun** [reply]
Super!

**Reply by jindradev** [reply]
omg I didn't realize either. I hope others are smarter. At least there is a search bar :) maybe better communication with people about how this community calls it would be nice. This leads me to a discussion I'm about to start - showing tags of the article you're about to read.
